package.json

```diff
@@ -1,222 +1,250 @@
+		"type": "git",
+		"url": "https://github.com/formulahendry/vscode-code-runner.git"
+	},
+	"activationEvents": [
+		"onCommand:code-runner.runCode",
+		"onCommand:code-runner.runCodeByLanguage",
+		"onCommand:code-runner.runCustomCommand"
+	],
+	"main": "./out/src/extension",
+	"contributes": {
+		"commands": [
+			{
+				"command": "code-runner.runCode",
+				"title": "Code Runner: Run Code"
+			},
+			{
+				"command": "code-runner.runCodeByLanguage",
+				"title": "Code Runner: Run Code By Language"
+			},
+			{
+				"command": "code-runner.runCustomCommand",
+				"title": "Code Runner: Run Custom Command"
+			},
+			{
+				"command": "code-runner.stop",
+				"title": "Code Runner: Stop Code Run"
+			}
+		],
+		"keybindings": [
+			{
+				"command": "code-runner.runCode",
+				"key": "ctrl+alt+n"
+			},
+			{
+				"command": "code-runner.runCodeByLanguage",
+				"key": "ctrl+alt+j"
+			},
+			{
+				"command": "code-runner.runCustomCommand",
+				"key": "ctrl+alt+k"
+			},
+			{
+				"command": "code-runner.stop",
+				"key": "ctrl+alt+m"
+			}
+		],
+		"menus": {
+			"editor/context": [
+				{
+					"when": "!inOutput",
+					"command": "code-runner.runCode",
+					"group": "navigation"
+				},
+				{
+					"when": "inOutput",
+					"command": "code-runner.stop",
+					"group": "stop-code-run"
+				}
+			]
+		},
+		"configuration": {
+			"type": "object",
+			"title": "Run Code configuration",
+			"properties": {
+				"code-runner.executorMap": {
+					"type": "object",
+					"default": {
+						"AppleScript": "osascript.exe",
+						"Batch File": "CMD.exe /C",
+						"C":   "gcc.exe \"$fullFilePath\" -o \"$fullFilePath.exe\" && \"$fullFilePath.exe\"",
+						"C++": "g++.exe \"$fullFilePath\" -o \"$fullFilePath.exe\" && \"$fullFilePath.exe\"",
+						"C#": "csc.exe /NoLogo out:\"$fullFilePath.exe\" \"$fullFilePath\" && \"$fullFilePath.exe\"",
+						"Clojure": "lein.exe exec",
+						"CoffeeScript": "coffee.exe",
+						"Crystal": "crystal.exe",
+						"Elixir": "elixir.exe",
+						"F#": "fsi.exe",
+						"Go": "go.exe run",
+						"Groovy": "groovy.exe",
+						"Java": "javac.exe \"$fullFilePath\" && java.exe \"$fullFilePath.jar\"",
+						"JavaScript": "cscript.exe //NoLogo",
+						"Julia": "julia.exe",
+						"LUA": "lua.exe",
+						"OCaml": "ocaml.exe",
+						"Perl": "perl.exe",
+						"PHP": "php",
+						"PowerShell": "powershell.exe -ExecutionPolicy ByPass -File",
+						"Python": "python.exe",
+						"R": "Rscript.exe",
+						"Ruby": "ruby.exe",
+						"Scala": "scala.exe",
+						"Shell Script (Bash)": "bash",
+						"Swift": "swift.exe",
+						"TypeScript": "ts-node.exe",
+						"VB Script": "cscript.exe //NoLogo",
+						"VB.NET": "vbc.exe /NoLogo out:\"$fullFilePath.exe\" \"$fullFilePath\" && \"$fullFilePath.exe\""
+					},
+					"description": "Set the executor of each language."
+				},
+				"code-runner.executorMapByFileExtension": {
+					"type": "object",
+					"default": {
+						".bat": "CMD.exe /C",
```

Comment:
No need to add executor for each file extension. It is what [Language Identifiers](https://code.visualstudio.com/docs/languages/identifiers) need to do.